project/Settings.scala

```diff
@@ -302,4 +315,8 @@ object Settings {
     }
   }
 
+  def emptyIfScala213(value: Option[String], scalaBinaryVersionString: String): Option[String] = {
```

Comment:
Thanks! One more thing this def emptyIfScala213 is no longer needed

Reply:
Thanks. Fixed that, just now.